dbms/src/Common/TargetSpecific.h

```diff
@@ -697,3 +726,85 @@ TIFLASH_TARGET_SPECIFIC_NAMESPACE(
 #undef DECLARE_TYPE
 #undef ENUM_TYPE
 #undef GET_TYPE
+
+// clang-format off
+/** Runtime Dispatch helpers for class members.
```

Comment:
Does it duplicate with the macro `TIFLASH_DECLARE_MULTITARGET_FUNCTION` above?

Reply:
TIFLASH_DECLARE_MULTITARGET_FUNCTION can not used for class member function due to `_Pragma`